scripts/lib/backlog.sh

```
@@ -0,0 +1,194 @@
+  if [ -z "$active_match" ] && [ -n "$done_match" ]; then
+    # Already archived — idempotent success.
+    return 0
+  fi
+  if [ -z "$active_match" ]; then
+    echo "backlog_archive: no ticket file found for ID '$id'" >&2
+    return 1
+  fi
+
+  mkdir -p "$root/backlog.d/_done"
+  (
+    cd "$root"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)